build.assets/build-webassets-if-changed.sh

```diff
@@ -56,14 +56,18 @@ for i in "${!SRC_DIRECTORIES[@]}"; do
 done
 
 # Calculate the current hash-of-hashes of the given source directories, package.json, and pnpm-lock.yaml.
-# We exclude node_modules as it's covered by package.json and pnpm-lock.yaml.
-# We also exclude .swc as it's a cache directory for the swc compiler,
-# and ironrdp/pkg as it's filled with the generated wasm files.
+# Excluded files and folders:
+# * node_modules: it's covered by package.json and pnpm-lock.yaml.
+# * .swc: it's a cache directory for the swc compiler.
+# * ironrdp/pkg: it's filled with the generated wasm files.
+# * web/.storybook: it contains public/mockServiceWorker.js which is autogenerated
```

Comment:
Should we ignore `web/.storybook/public/mockServiceWorker.js` specifically instead? Or is the whole of `.storybook` guaranteed to not be needed for webassets anyway?

Reply:
The whole directory is guaranteed to not be needed for webassets. It's for [Storybook](https://storybook.js.org/), which is a way to preview individual React components in specific states, without having to manipulate the state of a real app.